### ISSUE TITLE
Client crashes when sent packet 0x02.

### ISSUE THREAD

**Issue by mrpimpunicorn** [SirCmpwn/TrueCraft#221]

**Comment by SirCmpwn** [comment]
Really? That packet is essential to the connection handshake. You shouldn't even be able to log in if it causes crashes.

**Comment by mrpimpunicorn** [comment]
I don't think I am, I don't start loading chunks or anything. A frame or two is rendered (blue sky with empty hotbar) before the game crashes.

**Comment by SirCmpwn** [comment]
Weird. I'm able to play without any trouble. Restart the server?

**Comment by mrpimpunicorn** [comment]
I'm playing in singleplayer, I'll try to make a new world and see if that helps.

**Comment by SirCmpwn** [comment]
Can you get the stack trace?

**Comment by mrpimpunicorn** [comment]
No, I can't replicate it anymore, even with the old world. On another note, how can I access light level for a block (client-side)? Chunk.GetBlockLight seems to always return 0.

**Comment by SirCmpwn** [comment]
Well, you would use Chunk.GetBlockLight. But it's possible that it's not coming back correctly: https://github.com/SirCmpwn/TrueCraft/blob/master/TrueCraft.Client/Handlers/ChunkHandlers.cs#L41

Also possible that the value from the server is not correct. Use `/sl` to get the skylight at your current position (`/bl` does not exist but would be easy to add).
